components/Document/SubmissionDetails.tsx

```diff
@@ -35,29 +39,56 @@ function SubmissionDetails({
   const visibleHubs = hubs?.slice(0, sliceIndex) ?? [];
   const hiddenHubs = hubs?.slice(sliceIndex) ?? [];
 
+  let authorProfile = createdBy?.authorProfile;
+  let bounty: Bounty | undefined;
+  if (bounties.length > 0) {
+    // @ts-ignore
+    authorProfile = bounties[0].createdBy?.authorProfile;
+    bounty = bounties[0];
+  }
+
```

Comment:
Let's avoid variable assignments like these. It's really hard to read after some time passes. Also, pretty sure no need for ts-ignore
```suggestion
   const bounty =  (bounties ?? [])[0]?.createdBy?.authorProfile;
  }
```

Reply:
Thanks for the suggestion. It doesn't quite accomplish what I was trying to do (notice there are two vars assigned) but I get the sentiment of it and will update